src/shared/containers/AnalysisPlugin/AnalysisPluginContainer.tsx

```diff
@@ -284,10 +285,11 @@ const AnalysisPluginContainer = ({
           const imageId = asset.filePath.substring(
             asset.filePath.lastIndexOf('/') + 1
           );
-          const imgResource = (await nexus.Resource.get(
+
+          const fileResource = (await nexus.File.get(
             orgLabel,
             projectLabel,
-            encodeURIComponent(asset.filePath)
+            encodeURIComponent(imageId)
```

Comment:
`fileId` instead of `imageId`

Reply:
imageId is the uuid renaming it is not needed. it is also used elsewhere